tests/manage/monitoring/prometheus/test_capacity.py

```diff
@@ -44,16 +44,34 @@ def test_capacity_workload_alerts(
     alerts = workload_storageutilization_95p.get('prometheus_alerts')
     # TODO(fbalak): it seems that CephFS utilization triggers only firing
     # alerts. This needs to be more investigated.
+
+    if config.ENV_DATA.get('ocs_version') == '4.2':
```

Comment:
Would that work for both 4.2.0 and 4.2.1?

Reply:
For OCS 4.2 we have currently only this config:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocs_version/ocs-4.2.yaml